README.md

```diff
@@ -51,7 +51,6 @@ support today's systems.
 | **Feature**                 | **Status**
 |-                            |-
 | **CD-DA file codecs**       | Yes: Opus, OGG/Vorbus, MP3, FLAC, and WAV
-| **Pixel-perfect mode**      | Yes: `output=openglpp` or `output=texturepp`
```

Comment:
@GranMinigun , I don't think should commit a change to readme.md since it still applies to the latest release. We should wait to change the entire documentation in bulk for 0.81.

Reply:
Fair point, probably best to leave it for now.

Reply:
> Remove/update references to texturepp and openglpp in README and [README.md](http://readme.md/).

@johnnovak , I think we should wait before changing any of the documentation in the readme.md because it still applies for the latest release ... That's why for instance I held off adding Voodoo to the [README.md](http://readme.md/) and I believe @kcgen did not add IMFC (IBM Music Feature Card) there as well.

Reply:
Yeah but you get my point about updating everything else, right? All build instructions, etc. Those get updated whenever we change main.

Reply:
> Well, on a second thought, we're updating all the .md files in the main branch to reflect the "current dev" state (e.g. build instructions), so maybe it's ok to change it, after all? @kcgen? But the [README.md](http://readme.md/) everybody sees when goes to our repo is a bit special, admittedly.

@johnnovak Okay, so I had to look into my private messages with @kcgen just be sure I wasn't remembering things wrong. A couple of weeks ago, I asked @kcgen about updating the User Section of the README.md to add some of the new features, and he told me that updating the README.md was part of the release steps. And looking back, that makes 100% sense.

In my humble opinion, the README.md specifically should represent the state of the project at the latest stable release. Besides, it is better to review and discuss the changes bulk Pull Request with all the additions and removals to the sections than if we were to do it by small commits inside multiple PRs. In doing that, we may overload the front page with a lot of smaller features, or we may keep outdated information by mistake.

Reply:
At the end of the day, I hope I'm not being too pedantic about this. I'll side with whatever @kcgen and @dreamer decides to do about this. However, I feel like this should be another addition to the [CONTRIBUTING.md](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md), just to prevent this happening again.

Reply:
Made the PR https://github.com/dosbox-staging/dosbox-staging/pull/2470 so we can easily prevent stuff like this happening in the future.

Reply:
> > Well, on a second thought, we're updating all the .md files in the main branch to reflect the "current dev" state (e.g. build instructions), so maybe it's ok to change it, after all? @kcgen? But the [README.md](http://readme.md/) everybody sees when goes to our repo is a bit special, admittedly.
> 
> @johnnovak Okay, so I had to look into my private messages with @kcgen just be sure I wasn't remembering things wrong. A couple of weeks ago, I asked @kcgen about updating the User Section of the README.md to add some of the new features, and he told me that updating the README.md was part of the release steps. And looking back, that makes 100% sense.

Yes, it makes sense… Updating README.md *on the release branch*. That's also an opportunity to reassess if we haven't missed anything (as often happens). We might later cherry-pick that commit on the main branch, but it's much better to be proactive about it.

> In my humble opinion, the README.md specifically should represent the state of the project at the latest stable release.

Nope. This makes it harder for all developers. Documentation evolves with the code; if you see documentation not matching the code on the same branch, that means there is a bug. Inventing some additional rules about documentation on branch A representing project state on branch B makes everything unnecessarily complicated.

> Besides, it is better to review and discuss the changes in bulk, with all the additions and removals to the sections, than if we were to do it by small commits inside multiple PRs. (…)

Again, I do not agree - it's much better to have incremental changes (also in documentation), and then have separate PR to reedit them as necessary.

> In doing that, we may overload the front page with a lot of smaller features, or we may keep outdated information by mistake.

You have a point here - our README.md file again creeped out to be overloaded.

I think we no longer need "summary of features" table (at least not in README) - we have "Feature highlights" on https://dosbox-staging.github.io/ already. Perhaps we should replace this table in README.md with a new page on webpage listing *all* the features? @kcgen @johnnovak @ThomasEricB what do you think?

Reply:
In this context: I think "Pixel-perfect mode" line in table should not be removed, but rather edited to say something like:

> Pixel-perfect mode: Replaced by integer viewport scaling (link to discussions).

Reply:
> I think we no longer need "summary of features" table (at least not in README) - we have "Feature highlights" on https://dosbox-staging.github.io/ already. Perhaps we should replace this table in README.md with a new page on webpage listing all the features? @kcgen @johnnovak @ThomasEricB what do you think?

While we slightly disagree (as you think the "summary of features" should reflect main while I think it should reflect the stable release), I agree with your solution. I think we should purge that section and replace it with a new page on webpage listing all the features.

Reply:
Yes, with the documentation site, we can  soon separate out the  notable changes from the readme (which was becoming almost like a project changelog) and make it more succinct (project description and quick build notes). 

My hunch is that the documentation that describes all the exposed interface will capture our features.

Every line in the conf file : they come from SVN or we changed them (and will be documented). 

All the new z: programs. 

New features that may not land in the above, but need documenting (like the virtual-host mouse integration). 

That's going to be a lot of stuff, however, that table-of-contents will almost act like the readme table we have now, where users can click and read more about the feature. 

Except with the documentation site, the TOC will take them to real documentation instead of PRs.

Reply:
> But in the meantime - I agree we might as well add the current features to readme and make it reflect main. Sorry for the confusion I introduced, @ThomasEricB !

No problem. Relevant to @GranMinigun 's PR however, this still need changing before merging into main. To solve this, I think dreamer's solution is sensible: https://github.com/dosbox-staging/dosbox-staging/pull/2456#discussion_r1191266291

Reply:
Pretty much agree with all you wrote @kcgen and @dreamer.

TL;DR: 

- README.md should evolve with the current dev version and should reflect the state of main.
- Until the manual is not published, the feature table in READMe.md could be updated. Once the manual is out, the table is no longer needed and should be removed.
- The feature highlights on the front page of the website is not a _comprehensive_ list of all features, just a list of most notable features to "entice" users. It should and will not include everything (it should still look good in the design); that role will be filled by the documentation that will be comprehensive (e.g. a full expanded TOC, or perhaphs an index will effectively list all we have, as @kcgen rightly pointed out).

Reply:
So @GranMinigun , please update README.md with info about this feature.